src/eddymotion/estimator.py

```diff
@@ -86,8 +86,11 @@ def fit(
         """
         align_kwargs = align_kwargs or {}
 
+        _seed = None
         if seed or seed == 0:
-            np.random.seed(20210324 if seed is True else seed)
+            _seed = 20210324 if seed is True else seed
```

Comment:
While we're here, can you add the default behavior to the docstring? IIUC, the default is to use `20210324` as the seed?

Reply:
> While we're here, can you add the default behavior to the docstring? 

Done in d77d289.